aiidalab_widgets_base/viewers.py

```diff
@@ -993,7 +1014,8 @@ def _observe_selection_adv(self, _=None):
 
     @observe("selection")
     def _observe_selection_2(self, _=None):
-        self.selection_info.value = self.create_selection_info()
+        if hasattr(self, "selection_info"):
```

Comment:
I am not happy about this, but since this observer is in `StructureDataViewer` class, I cannot register it inside `_selection_tab`. Ideas welcomed. :blush:

Reply:
As I wrote in the other comments, `selection_info` should always exist. So there is, probably, no need to add an `if` statement here.